NBitcoin.TestFramework/WellknownNodeDownloadData.cs

```diff
@@ -434,6 +434,35 @@ public class GroestlcoinNodeDownloadData
 			};
 		}
 
+                public class BitcoreNodeDownloadData
```

Comment:
wrong indentation, should be tabs, not spaces

Reply:
Changed